### ISSUE TITLE
[Question] Is the API now reasonably stable?

### ISSUE THREAD

**Issue by Sophrinix** [tidev/ioslib#9]
Hey @ingo  @cb1kenobi @jhaynie are there any major planned API changes for this library. 

I've started the process of updating hyperloop-ios/lib/library.js to use the new library. 
I'm driving the upgrade via the unit test ( really happy to see them there).

I just wanted to make sure that no additional major breakage is likely until Developer Preview 2 comes out. I get the "why". I just making sure that I'm not being prematurely too ambitious.

**Comment by hbugdoll** [comment]
In my opinion, this issue can also be closed 🦕 .

**Comment by cb1kenobi** [comment]
Yes, the ioslib API is very stable. It will likely never change. Closing.
